packages/language/src/language-server/hover-request.ts

```diff
@@ -17,7 +17,7 @@ import {
   isNameToken,
   isReferenceToken,
 } from "../linking/tokens";
-import { Token } from "../parser/tokens";
+import * as tokens from "../parser/tokens";
```

Comment:
Minor: I believe in most other files we simply use `t`. We can simply follow the convention introduced by the other files.
```suggestion
import * as t from "../parser/tokens";
```